Gemfile

```diff
@@ -37,6 +37,9 @@ gem 'countries'
 #select country info in the ISO standard
 gem 'country_select'
 
+#parse html
+gem 'nokogiri', '~>1.6.8'
```

Comment:
Thanks, you're right, I read it too fast and jumped to the dependencies section 🙊